lbrynet/tests/unit/dht/test_messages.py

```diff
@@ -13,36 +13,44 @@
 class DefaultFormatTranslatorTest(unittest.TestCase):
     """ Test case for the default message translator """
     def setUp(self):
-        self.cases = ((RequestMessage('1' * 48, 'rpcMethod',
+        self.cases = ((RequestMessage('1' * 48, '127.0.0.1', 4444, 'rpcMethod',
```

Comment:
@kaykurokawa do we already have some function to pack an IP address in binary form instead of sending the whole string? I guess we can just do that with some python primitive?

Reply:
I am converting IP from str to binary form using:  ```[int(x) for x in message.nodeIP.split('.')]``` and then back with ```".".join([str(d) for d in msgPrimitive[self.headerNodeIP][:4]])```